web/src/app/services/AlertMetrics/AlertMetrics.tsx

```diff
@@ -83,6 +83,8 @@ export default function AlertMetrics({
         notCreatedBefore: since.toISO(),
         createdBefore: until.toISO(),
         filterByStatus: ['StatusClosed'],
+        notClosedBefore: since.toISO(),
+        closedBefore: until.toISO(),
```

Comment:
Do we need to include this in a separate `useQuery` call since we're also setting `createdBefore` and `notCreatedBefore` here? cc @KatieMSB

Reply:
Good point, we probably actually don't need to set `createdBefore` and `notCreatedBefore` anymore since we only care about closed alerts for metrics.